registrations/serializers.py

```diff
@@ -61,6 +49,68 @@ def create(self, validated_data):
         else:
             raise DRFPermissionDenied("The waiting list is already full")
 
+    def validate(self, data):
+        errors = {}
+        registration = data["registration"]
+        mandatory_fields = {mf.id for mf in registration.mandatory_fields.all()}
+
+        mandatory_field_mapping = [
+            {
+                "field_ids": ["street_address"],
+                "mandatory_field_id": MandatoryField.DefaultMandatoryField.ADDRESS,
+            },
+            {
+                "field_ids": ["city", "zipcode"],
+                "mandatory_field_id": MandatoryField.DefaultMandatoryField.CITY,
```

Comment:
Is this mapping mandatory? Could we just have `MandatoryField.DefaultMandatoryField.ZIPCODE`?

Reply:
Yeah, I could simplify the logic and forget the mapping. After that only checking that mandatory field id would match the field name. Also need to rename address mandatory field as street_address in that case